python/erdos/message.py

```diff
@@ -57,7 +59,7 @@ def __str__(self):
         return "{{timestamp: {}, data: {}}}".format(self.timestamp, self.data)
 
 
-class WatermarkMessage(Message):
+class WatermarkMessage(Message[T]):
```

Comment:
```suggestion
class WatermarkMessage(Message[None]):
```
The `WatermarkMessage` is a special type of the `Message` class where the data being carried is empty, so we annotate that accordingly.